docs/guides/go-built-in.md

```diff
@@ -471,8 +449,8 @@ execution of the block once accepted.
 In the following code, the application simply returns the unmodified group of transactions:
 
 ```go
-func (app *KVStoreApplication) PrepareProposal(proposal abcitypes.RequestPrepareProposal) abcitypes.ResponsePrepareProposal {
-	return abcitypes.ResponsePrepareProposal{Txs: proposal.Txs}
+func (app *KVStoreApplication) PrepareProposal(_ context.Context, proposal *abcitypes.RequestPrepareProposal) (*abcitypes.ResponsePrepareProposal, error) {
```

Comment:
In line 446, above, where you see "does not use more bytes than `RequestPrepareProposal.max_tx_bytes'" you should have "does not use more bytes than `RequestPrepareProposal.max_tx_bytes`" with a backtick instead of single quotes at the end.

Reply:
Ah! Good catch. Thanks.